src/main/java/com/example/finance/listener/BudgetEntityListener.java

```diff
@@ -0,0 +1,46 @@
+package com.example.finance.listener;
+
+import com.example.finance.model.entity.BudgetEntity;
+import jakarta.persistence.*;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+@Component
+public class BudgetEntityListener {
```

Comment:
That listener was developed only for our meeting purpose. If you don't have any reason to keep it, delete it :)